src/algo/DequeArray.js

```diff
@@ -150,6 +150,8 @@ export default class DequeArray extends Algorithm {
 		this.front = 0;
 		this.size = 0;
 		this.leftoverLabelID = this.nextIndex++;
+		this.leftoverValID = this.nextIndex++;
```

Comment:
There's a similar issue here to the thing I mentioned with LCS:

In `reset()` on line 234 the value of `nextIndex` is increased by 7 to avoid overwriting the IDs of any labels created during `setup()`. Since you're adding these two label IDs here, you'll need to change "7" to "9" on that line to avoid any object creation/deletion issues.

I would leave a comment on that line but GitHub only allows inline comments on changed lines :/